installer/ansible/group_vars/auth.yml

```diff
@@ -0,0 +1,38 @@
+# Copyright 2019 The OpenSDS Authors.
```

Comment:
SODA Authors

Reply:
Why do we need all these files for single project installer also...(separate files in group_vars)

Reply:
Removed